<launch>

    <node name="map_processing_node" pkg="coverage_boustrophedon" type="map_processing_node" output="screen">
        <param name="inflation_radius" value="2.0" />
    </node>

    <node name="coverage_node" pkg="coverage_boustrophedon" type="coverage_node" output="screen">
        <param name="tile_resolution" value="3" />
        <param name="goal_tolerance" value="1.5" />
    </node>

    <node name="simple_dubins_path_node" pkg="coverage_boustrophedon" type="simple_dubins_path_node" output="screen">
        <param name="turning_radius" value="1.5" />
        <param name="path_resolution" value="0.05" />
    </node>

</launch>
